tools/occlum

```diff
@@ -443,6 +476,9 @@ case "$cmd" in
     status)
         cmd_status
         ;;
+    check)
+        cmd_check
```

Comment:
Agree with @guzongmin . Useful feature, but not important enough to qualify as a command.

Instead of asking users to learn a new command, I think a more desired user experience is to do this check behind the scene automatically: before every `occlum run` or `occlum exec` command, run the `detect_fsgsbase` tool. If found any problem, output error messages to the user. If it is ok, save the success result to some location so that the tool does not need to be run again for the same Occlum instance.

Reply:
Thank you for your review. Move this check to `occlum run/start`